docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_an_databricks_spark_cluster.rst

```diff
@@ -0,0 +1,167 @@
+               "base_directory": "REPLACE ME",  # TODO: replace with your value
+           },
+       },
+       "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+    },
+    expectations_store_name="expectations_store",
+    validations_store_name="validations_store",
+    evaluation_parameter_store_name="evaluation_parameter_store",
+    data_docs_sites={
+       "local_site": {
+           "class_name": "SiteBuilder",
+           "store_backend": {
+               "class_name": "TupleFilesystemStoreBackend",
+               "base_directory": "REPLACE ME",  # TODO: replace with your value
+           },
+           "site_index_builder": {
+               "class_name": "DefaultSiteIndexBuilder",
+               "show_cta_footer": True,
+           },
+       }
+    },
+    validation_operators={
+       "action_list_operator": {
+           "class_name": "ActionListValidationOperator",
+           "action_list": [
+               {
+                   "name": "store_validation_result",
+                   "action": {"class_name": "StoreValidationResultAction"},
+               },
+               {
+                   "name": "store_evaluation_params",
+                   "action": {"class_name": "StoreEvaluationParametersAction"},
+               },
+               {
+                   "name": "update_data_docs",
+                   "action": {"class_name": "UpdateDataDocsAction"},
+               },
+           ],
+       }
+    },
+    anonymous_usage_statistics={
+     "enabled": True
+    }
+    )
+
+   context = BaseDataContext(project_config=project_config)
+
+#. **Install Great Expectations on your Databricks Spark cluster.**
+
+   Copy this code snippet into a cell in your Databricks Spark notebook and run it:
+
+   .. code-block:: python
+
+      dbutils.library.installPyPI("great_expectations")
+
+
+#. **Configure an Expectation store in DBFS**
+
+   Replace the "REPLACE ME" on lines 27 of the code snippet with the path to your Expectation Store on DBFS.
+
+   .. code-block:: python
+
+      path_to_expectation_store =  "/dbfs/FileStore/expectations/"
+
+#. **Configure an Validation Result store in DBFS.**
+
+   Replace the "REPLACE ME" on lines 34 of the code snippet with the path to your Validation Store on DBFS.
+
+   .. code-block:: python
+
+      path_to_validation_store =  "/dbfs/FileStore/validations/"
+
+
+#. **Configure an Data Docs website in DBFS.**
+
+   Replace the "REPLACE ME" on line 47 of the code snippet with the path to your DataDocs Store on DBFS.
+
+   .. code-block:: python
+
+      path_to_datadocs_store =  "/dbfs/FileStore/docs/"
+
+
+#. **Test your configuration.**
+
+   Execute the cell with the snippet above.
+
+   Then copy this code snippet into a cell in your Databricks Spark notebook, run it and verify that no error is displayed:
+
+   .. code-block:: python
+
+      context.list_datasources()
+
+
+Additional notes
+----------------
+
```

Comment:
The original `Additional Notes` section included

```

 # File location and type
file_location = "/FileStore/tables/dc_wikia_data.csv"
file_type = "csv"

# CSV options
infer_schema = "false"
first_row_is_header = "false"
delimiter = ","

# The applied options are for CSV files. For other file types, these will be ignored.
   df = spark.read.format(file_type) \
     .option("inferSchema", infer_schema) \
     .option("header", first_row_is_header) \
     .option("sep", delimiter) \
     .load(file_location)


- Once loaded, `SparkDFDataset` using the loaded Dataframe and DataContext that we created in the above section. 


	GE_spark_df_dataset = SparkDFDataset(df, data_context=context)
	# expectations can now be run using the Dataset
	GE_spark_df_dataset. expect_table_row_count_to_equal(100)

```

Reply:
note : change the `SparkDFDataset(df, data_context=context)` --> get_batch.batch_kwargs(dataset:df)